src/app/options.cpp

```diff
@@ -247,6 +315,13 @@ namespace
     constexpr const StringOption CONFIGURATION_OPTION = {"configuration"};
     constexpr const BoolOption PORTABLE_OPTION = {"portable"};
     constexpr const BoolOption RELATIVE_FASTRESUME = {"relative-fastresume"};
+    constexpr const StringOption SAVE_PATH_OPTION = {"save-path"};
+    constexpr const TriStateBoolOption PAUSED_OPTION = {"add-paused"};
+    constexpr const BoolOption SKIP_HASH_CHECK_OPTION = {"skip-hash-check"};
+    constexpr const StringOption CATEGORY_OPTION = {"category"};
+    constexpr const BoolOption SEQUENTIAL_OPTION = {"sequential"};
+    constexpr const BoolOption FIRST_AND_LAST_OPTION = {"first-and-last"};
```

Comment:
@glassez You're right, it doesn't have an effect in that case. I'm not sure why, but the issue seems to be somewhere within `BitTorrent::Session::addTorrent()` and/or everything it calls into. I don't think I'm the right person to debug this one.